src/redux/categories/categories.js

```diff
@@ -0,0 +1,27 @@
+// action types
+
+export const SHOW_STATUS = 'categories/categories/SHOW_STATUS';
+
+// action creators
+
+export const showStatus = () => (dispatch) => {
+  dispatch({
+    type: SHOW_STATUS,
+    payload: ['under construction'],
+  });
```

Comment:
- [ ] Kindly understand the requirement that it should return a string not an array. Kindly change this with :arrow_down:
```
type: SHOW_STATUS,
payload: 'under construction',
```

Reply:
resolved